src/index.js

```diff
@@ -82,17 +82,53 @@ class Purgecss {
      */
     purge() {
         // Get selectors from content files
-        let cssClasses = this.extractFileSelector(this.options.content, this.options.extractors)
+        const content = this.options.content
+
+        const cssFileClasses = this.extractFileSelector(content.filter(o => typeof o === 'string'), this.options.extractors)
+        const cssRawClasses = this.extractRawSelector(content.filter(o => typeof o === 'object'), this.options.extractors)
+
         // Get css selectors and remove unused ones
-        let files = []
-        for (let file of this.options.css) {
-            const cssContent = this.options.stdin ? file : fs.readFileSync(file, 'utf8')
-            files.push({
+        const sources = this.getCssContents(this.options.css, new Set([...cssFileClasses, ...cssRawClasses]))
+
+        return sources
+    }
+
+    getCssContents(cssOptions: Array<any>, cssClasses: Set<string>) {
+        const sources = []
+        cssOptions.forEach(option => {
+            let file = null
```

Comment:
use `for...of` instead of `forEach`, it stays consistent with `extractFileSelector`